test/test_videoapi.py

```diff
@@ -299,6 +299,14 @@ def test_keyframe_reading(self, test_video, config, backend):
                 for i in range(len(av_keyframes)):
                     assert av_keyframes[i] == approx(vr_keyframes[i], rel=0.001)
 
+    def test_src(self):
+        with pytest.raises(ValueError, match="src cannot be empty"):
+            VideoReader(src="")
+        with pytest.raises(ValueError, match="`src` must be either string"):
```

Comment:
If removing ticks from the message:
```suggestion
        with pytest.raises(ValueError, match="src must be either string"):
```